examples/adc.rs

```diff
@@ -13,7 +13,7 @@ async fn main() -> TockResult<()> {
     let mut driver = context.create_timer_driver()?;
     let timer_driver = driver.activate()?;
 
-    let mut console = Console::new();
+    let mut console: Console = Default::default();
```

Comment:
Is this a `clippy` lint? If yes, if this is a transitional state, maybe `clippy` can be switched off for `Default` `impl`s?

Reply:
I now replaced `Console::new()` by `Console::default()`. I think that the lint actually makes sense (now that I think of it) as there is a distinguished way to create default values and this is `Default::default()` and so you should implement it (so others can use it).